jetstream/bigquery_client.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+import re
+import time
+from typing import Optional
+
+import attr
+import google.cloud.bigquery.client
+import google.cloud.bigquery.dataset
+import google.cloud.bigquery.job
+import google.cloud.bigquery.table
+from google.cloud.bigquery_storage_v1beta1 import BigQueryStorageClient
+
+
+@attr.s(auto_attribs=True, slots=True)
+class BigQueryClient:
```

Comment:
`Analysis` is getting quite long. I think it makes sense to separate this one out, also because now it's not only used in `Analysis` anymore